src/inputs/select/select.jsx

```diff
@@ -79,6 +79,11 @@ const propTypes = {
      * A Select can have multiple values
      */
     multiple: PropTypes.bool,
+    /**
+     * Placeholder displayed when there are no matching search results or a falsy value to hide it
+     * (can also be a react component)
+     */
+    noResultsText: PropTypes.string,
```

Comment:
> or a falsy value to hide it
> (can also be a react component)

Do we maybe need `PropTypes.oneOfType[]` here with some different options?

Reply:
Yeah, you are right. Their docs were wrong.

Reply:
Fixed

Reply:
So that will allow a component or a string.  By "a falsy value to hide it" I presume we mean an empty string? ;-)

Reply:
Should we fork 'em and submit a PR to fix their docs? 😁

Reply:
Hahaha! Actually it's a good idea, but I am sure because we're on v1.3.0 they going to say "get that corn out of my face."